src/GraphQL.Tests/Types/ComplexGraphTypeTests.cs

```diff
@@ -162,15 +162,63 @@ public void auto_register_input_object_graph_type()
 
     [Fact]
     public void accepts_property_expressions()
+    {
+        bool old = GlobalSwitches.InferFieldNullabilityFromNRTAnnotations;
+        GlobalSwitches.InferFieldNullabilityFromNRTAnnotations = false;
```

Comment:
TODO: move to `"StaticTests"`

Reply:
lol - i checked already - `ComplexGraphTypeTests` is already marked with `[Collection("StaticTests")]`